core/stateful_precompile_test.go

```diff
@@ -339,19 +348,15 @@ func TestContractNativeMinterRun(t *testing.T) {
 			},
 			suppliedGas: precompile.MintGasCost,
 			readOnly:    false,
-			setupState: func(state *state.StateDB) {
-				precompile.SetContractNativeMinterStatus(state, adminAddr, precompile.AllowListAdmin)
-			},
-			expectedErr: precompile.ErrCannotMint.Error(),
+			expectedRes: []byte{},
 			assertState: func(t *testing.T, state *state.StateDB) {
 				res := precompile.GetContractNativeMinterStatus(state, adminAddr)
 				assert.Equal(t, precompile.AllowListAdmin, res)
```

Comment:
should we also remove setupStates for the `TestContractDeployerAllowListRun` tests (above)?